__tests__/app.test.js

```diff
@@ -122,3 +122,18 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("should respond with a status code:200 and a narray of user objects with the properties: username, name, avatar_url", async () => {
+    const users = await request(app).get("/api/users");
+    console.log(users.body);
```

Comment:
please delete console logs when finished with to keep code clean :)

Reply:
ah sorry, missed that one!